src/IIIFPresentation/API.Tests/Integration/ModifyManifestExternalItemsTests.cs

```diff
@@ -0,0 +1,1257 @@
+        canvasPainting1.CanvasOrder.Should().Be(0);
+        canvasPainting1.ChoiceOrder.Should().Be(1);
+        canvasPainting1.StaticWidth.Should().Be(617);
+        canvasPainting1.StaticHeight.Should().Be(1024);
+        
+        var canvasPainting2 = canvasPaintings.Last();
+        canvasPainting2.Id.Should().Be(canvasPainting1.Id, "Canvas choices share same id");
+        canvasPainting2.CanvasOriginalId.Should().Be("https://iiif.example/CreateManifest_ExternalItemsWithChoices_CreatedDBRecord.json");
+        canvasPainting2.CanvasOrder.Should().Be(0, "Canvas choices share same order");
+        canvasPainting2.ChoiceOrder.Should().Be(2);
+        canvasPainting2.StaticWidth.Should().Be(617);
+        canvasPainting2.StaticHeight.Should().Be(1024);
+    }
+    
+    [Fact]
+    public async Task PutFlatId_Insert_ExternalItems_ReturnsManifest()
+    {
+        // Arrange
+        var slug = $"slug_{nameof(PutFlatId_Insert_ExternalItems_ReturnsManifest)}";
+        var id = nameof(PutFlatId_Insert_ExternalItems_ReturnsManifest);
+        var manifest = $@"
+{{
+    ""@context"": ""http://iiif.io/api/presentation/3/context.json"",
+    ""id"": ""https://iiif.example/manifest.json"",
+    ""type"": ""Manifest"",
+    ""parent"": ""{RootCollection.Id}"",
+    ""slug"": ""{slug}"",
+    ""items"": [
+        {{
+            ""id"": ""https://iiif.example/{slug}.json"",
+            ""type"": ""Canvas"",
+            ""height"": 1800,
+            ""width"": 1200,
+            ""items"": [
+                {{
+                    ""id"": ""https://iiif.io/api/cookbook/recipe/0001-mvm-image/page/p1/1"",
+                    ""type"": ""AnnotationPage"",
+                    ""items"": [
+                        {{
+                            ""id"": ""https://iiif.io/api/cookbook/recipe/0001-mvm-image/annotation/p0001-image"",
+                            ""type"": ""Annotation"",
+                            ""motivation"": ""painting"",
+                            ""body"": {{
+                                ""id"": ""http://iiif.io/api/presentation/2.1/example/fixtures/resources/page1-full.png"",
+                                ""type"": ""Image"",
+                                ""format"": ""image/png"",
+                                ""height"": 1800,
+                                ""width"": 1200
+                            }},
+                            ""target"": ""https://iiif.io/api/cookbook/recipe/0001-mvm-image/canvas/p1""
+                        }}
+                    ]
+                }}
+            ]
+        }}
+    ]
+}}";
+        
+        var requestMessage =
+            HttpRequestMessageBuilder.GetPrivateRequest(HttpMethod.Put, $"{Customer}/manifests/{id}", manifest);
+        
+        // Act
+        var response = await httpClient.AsCustomer(1).SendAsync(requestMessage);
```

Comment:
Would it be worth defining the customer's ID as a constant value shared between the tests?

Reply:
Default of `.AsCustomer()` is 1 so will remove.